src/shmem_internal.h

```diff
@@ -384,9 +384,11 @@ extern shmem_internal_mutex_t shmem_internal_mutex_alloc;
 #   define SHMEM_MUTEX_DESTROY(_mutex)
 #   define SHMEM_MUTEX_LOCK(_mutex)
 #   define SHMEM_MUTEX_UNLOCK(_mutex)
-
+typedef uint64_t shmem_atomic_uint64_t;
```

Comment:
Delete me

Reply:
```
In file included from synchronization_c.c:28:0:
../../src/shmem_atomic.h:147:18: warning: redefinition of typedef ‘shmem_atomic_uint64_t’ [-Wpedantic]
 typedef uint64_t shmem_atomic_uint64_t;
                  ^
In file included from synchronization_c.c:27:0:
../../src/shmem_internal.h:387:18: note: previous declaration of ‘shmem_atomic_uint64_t’ was here
 typedef uint64_t shmem_atomic_uint64_t;
                  ^
  CC       collectives_f.lo
In file included from ../../src/shmem_synchronization.h:19:0,
                 from ../../src/shmem_collectives.h:19,
                 from collectives_f.c:32:
../../src/shmem_atomic.h:147:18: warning: redefinition of typedef ‘shmem_atomic_uint64_t’ [-Wpedantic]
 typedef uint64_t shmem_atomic_uint64_t;
                  ^
In file included from collectives_f.c:30:0:
../../src/shmem_internal.h:387:18: note: previous declaration of ‘shmem_atomic_uint64_t’ was here
 typedef uint64_t shmem_atomic_uint64_t;
```

Reply:
Hmm... Travis/Ubuntu gcc doesn't like this when I remove it (and no threading), which is probably why I threw a second typedef in there.  I'll figure something out.